change-notes/1.20/analysis-javascript.md

```diff
@@ -43,6 +43,7 @@
 | Assignment to exports variable             | Fewer results                | This query no longer flags code that is also flagged by the query "Useless assignment to local variable". |
 | Client-side cross-site scripting           | More true positive and fewer false positive results. | This query now recognizes WinJS functions that are vulnerable to HTML injection. It no longer flags certain safe uses of jQuery, and recognizes custom sanitizers. |
 | Hard-coded credentials                     | Fewer false positive results | This query no longer flags the empty string as a hardcoded username. |
+| Incomplete string escaping or encoding | Fewer false-positive results | This rule no longer flags calls that unwraps a string from a pair of delimiters. |
```

Comment:
Change notes for patch releases don't go here, I think; this should go on the JIRA issue instead.

Also: "unwraps" is singular, but "calls" is plural.

Reply:
Fixed. Added change note to specialized ticket: https://jira.semmle.com/browse/ODASA-7907.